yandex_contest/bin_sum.py

```diff
@@ -0,0 +1,53 @@
+def convert(string):
+    list_1 = []
+    list_1[:0] = string
+    list_x = [int(i) for i in list_1]
+    return list_x
+
+def main():
+    num_1 = input()
```

Comment:
There is some inconsistency in the naming of variables, eg `num_1` vs `number1`. Not clear what's the difference, why one is named with `_` and another -- without.

Let's have some self-explainatory names, eg something like `num_1` -> `binary_num1`.
`number_1` is not a number at all, it's a list, so `reversed_num_1` can be a good name